docs/source/examples.normal_modes.md

```diff
@@ -14,14 +14,14 @@ from wfl.configset import ConfigSet, OutputSpec
 from wfl.generate import normal_modes as nm 
 
 mols = [molecule("CH4"), molecule("H2O")]
-ConfigSet = ConfigSet(mols)
+configset = ConfigSet(mols)
 OutputSpec = OutputSpec("molecules.normal_modes.xyz")
```

Comment:
like `configset`, this one should also not redefine the symbol for the class itself

Reply:
slipped through, just fixed.